### DISCUSSION TITLE
Add count to tab view for unanswered

### DISCUSSION THREAD

**Discussion by wsmelton** [community/community#2909] [Discussions] [answered]
Issues and Pull requests tab we have a view of open items. It would be nice if Discussions tab offered the same view so we know how many items are there for unanswered questions.

**Comment by evi-liu** [comment, marked as answer]
Thanks for the feedback! In the early days of Discussions beta, we had a count on the tab. However, maintainers provided feedback that the count actually induced stress to keep that number at 0. The goal of Discussions is to relieve maintainer burden by allowing the community to step in and answer each other. Thus, we decided to remove that count. 

Instead seeing the exact count of unanswered, **would seeing an indicator for any new Discussions since you last visited meet your scenario here?**

**Reply by wsmelton** [reply]
Yes, that would help.

**Reply by evi-liu** [reply]
Sounds good. We are tracking a request for an activity indicator and will prioritize against the rest of our backlog.

**Reply by MiguelRipoll23** [reply]
Seeing an indicator for any new discussions since your last visited would work for me (or the pinned ones), thanks for all the work you guys are doing.

**Reply by yannickperrenet** [reply]
For me it is a showstopper that there is no number indicating activity for discussions, especially since it is a new feature. I would like to start using the discussions, but I am afraid no one will visit the discussions page (and thus making the work not worthwhile) because it draws no attention to itself.

**Reply by aschrijver** [reply]
New discussion indicator would be helpful. Right now there is zero attention to the Discussions tab, especially for projects that just started using them and visitors / contributors aren't really aware. Created a long discussion post with formatting and many links yesterday, but now feel like I have to also file an Issue to provide the link to it.

**Reply by windowsair** [reply]
I think there should be at least a switch to enable this feature...

**Reply by cx1111** [reply]
Yes, why not allow the repository owners to turn the count on or off in the settings, just like they already do with the discussions section itself?

It's highly confusing having tabs like issues and PRs with badge counts, and discussions without. It really makes the discussions section look barren.
